spec/views/page_template_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.feature "Page Template", type: :feature do
+  scenario 'Will have a footer on home page' do
+    visit '/'
+    expect(page.source).to include("<footer>")
+    expect(page.source).to include("</footer>")
```

Comment:
Perhaps an even better footer test:

```ruby
expect(page.source).to have_css('footer', text: "Made with love by the #April2017")
```

BUT since @ffasolin and @bruxelles86 are working on this now — let's go ahead and get this merged and they can add the test!